Common/CPUDetect.cpp

```diff
@@ -65,6 +65,7 @@ static unsigned long long _xgetbv(unsigned int index)
 #endif
 
 #if !defined(MIPS)
+#if !( defined(_WIN32) && defined(_M_ARM) )
```

Comment:
ARM simply should not compile this file.  It should instead only compile ArmCPUDetect.cpp.

-[Unknown]